dataactcore/models/lookups.py

```diff
@@ -0,0 +1,90 @@
+# This file defines a series of constants that represent the values used in the
+# broker's "helper" tables. Rather than define the values in the db setup scripts
+# and then make db calls to lookup the surrogate keys, we'll define everything
+# here, in a file that can be used by the db setup scripts *and* the application
+# code.
+# todo: replace getIdFromDict and getNameFromDict baseInterface functions with these constants
+
+from collections import namedtuple
+
+LookupType = namedtuple('LookupType', ['id', 'name', 'desc'])
+LookupFileType = namedtuple('LookupFileType', ['id', 'name', 'desc', 'letter', 'order'])
+
+FILE_STATUS = []
+FILE_STATUS.append(LookupType(1, 'complete', 'File has been processed'))
+FILE_STATUS.append(LookupType(2, 'header_error', 'The file has errors in the header row'))
+FILE_STATUS.append(LookupType(3, 'unknown_error', 'An unknown error has occurred with this file'))
+FILE_STATUS.append(LookupType(4, 'single_row_error', 'Error occurred in job manager'))
+FILE_STATUS.append(LookupType(5, 'job_error', 'File has not yet been validated'))
+FILE_STATUS.append(LookupType(6, 'incomplete', 'File has not yet been validated'))
```

Comment:
Obviously no big deal, but just wondering, why use a bunch of `append`s as opposed to defining the list all at once?